pom.xml

```diff
@@ -55,161 +68,89 @@
             <url>https://repo.jenkins-ci.org/snapshots/</url>
         </snapshotRepository>
     </distributionManagement>
-    <properties>
-        <jenkins.version>1.596.1</jenkins.version>
-        <workflow.version>1.7</workflow.version>
-    </properties>
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>1.16.1</version>
+            <version>2.3.5</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-step-api</artifactId>
-            <version>${workflow.version}</version>
+            <version>2.22</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>${workflow.version}</version>
+            <version>2.80</version>
```

Comment:
To simplify using test dependencies, you may consider using the Plugin BOM. https://github.com/jenkinsci/bom . It includes all Pipeline plugins and saves time on keeping versions aligned